pyproject.toml

```diff
@@ -97,7 +97,7 @@ upstream = [
     'fsspec @ git+https://github.com/fsspec/filesystem_spec',
     's3fs @ git+https://github.com/fsspec/s3fs',
     'kerchunk @ git+https://github.com/fsspec/kerchunk',
-    'icechunk @ git+https://github.com/earth-mover/icechunk#subdirectory=icechunk-python',
+    #'icechunk @ git+https://github.com/earth-mover/icechunk#subdirectory=icechunk-python',
```

Comment:
why remove icechunk?

Reply:
Oh that's some BS I have to do locally to get my tests to run, else I get some rust build error...

It started after there was a dodgy icechunk release, and I've just been living with it instead of fixing it 😅 But I didn't mean to commit it here so will remove it. (Yes I should probably just use pixi)